isos/base.sh

```diff
@@ -56,23 +56,22 @@ fi
 # prep the build system
 ensure_apt_packages cpio rpm tar ca-certificates xz-utils
 
-PKGDIR=$(mktemp -d)
+PKGDIR="/build"
 
 # initialize the bundle
 initialize_bundle $PKGDIR
 
 # base filesystem setup
 mkdir -p $(rootfs_dir $PKGDIR)/{etc/yum,etc/yum.repos.d}
 ln -s /lib $(rootfs_dir $PKGDIR)/lib64
-cp $DIR/base/*.repo $(rootfs_dir $PKGDIR)/etc/yum.repos.d/
```

Comment:
I'd be surprised if this didn't break things for local build where it's not being run from the Dockerfile